nx/test/nx_test.exs

```diff
@@ -1628,4 +1628,69 @@ defmodule NxTest do
                )
     end
   end
+
+  if Version.match?(System.version(), ">= 1.13.0-dev") do
+    describe "sigil" do
+      test "evaluates to tensor" do
+        import Nx
+
+        assert ~M[-1 2 3 4] == Nx.tensor([-1, 2, 3, 4])
+        assert ~M[1
+                  2
+                  3
+                  4] == Nx.tensor([[1], [2], [3], [4]])
+        assert ~M[1.0 2  3
+                  11  12 13] == Nx.tensor([[1.0, 2, 3], [11, 12, 13]])
+      end
+
+      test "evaluates with proper type" do
+        defmodule Test do
+          import Nx
+
+          def case1 do
+            unquote(Code.string_to_quoted!("~M[1 2 3 4]f32"))
```

Comment:
The unquote should not be needed here.

Reply:
It won't compile on `1.12` then:

```elixir
== Compilation error in file test/nx_test.exs ==
** (SyntaxError) test/nx_test.exs:1688:29: syntax error before: "8"
    (elixir 1.12.0) lib/kernel/parallel_compiler.ex:428: Kernel.ParallelCompiler.require_file/2
    (elixir 1.12.0) lib/kernel/parallel_compiler.ex:321: anonymous fn/4 in Kernel.ParallelCompiler.spawn_workers/7
```

Reply:
I see. So the whole test can be `Code.eval_string("import Nx; ~M[1 2 3 4]f32")`. You don't need the wrapper module. :)